### PR TITLE
fix: adding GlobalSecondaryIndexes to schema. test improvements

### PR DESCRIPTION
**Pull request by KodiVerse**

### CODE REVIEW

File: src/module/dy/validate-kwargs.js [hunk 1]
```diff
@@ -59,5 +57,5 @@ module.exports = (kwargs) => {
   if (result.error) {
     throw new Error(result.error);
   }
-  return result;
+  return result.value;
```
Comment (KodiVerse) [inline src/module/dy/validate-kwargs.js:60]: There was an issue in create-model with this. Made some improvements to the tests below.

File: test/module/dy.spec.js__cassettes/testingDyUtil_testingBasicLogic_recording.json [hunk 2]
```diff
@@ -25,6 +25,24 @@
           "KeyType": "RANGE"
         }
       ],
+      "GlobalSecondaryIndexes": [
```
Comment (KodiVerse) [inline test/module/dy.spec.js__cassettes/testingDyUtil_testingBasicLogic_recording.json:28]: Just re-recorded this test.

File: test/module/dy.spec.js [hunk 3]
```diff
@@ -27,8 +27,8 @@ describe('Testing dy Util', { useNock: true, nockStripHeaders: true }, () => {
       indices: {
         targetIndex: {
           // todo: this should error the way that is is
```
Comment (simlu) [inline test/module/dy.spec.js:29]: rm comment and add test

File: src/module/dy/create-model.js [hunk 4]
```diff
@@ -47,9 +54,6 @@ module.exports = (kwargs) => {
     ...(sortKey === undefined ? [] : [sortKey])
   ];
 
-  // todo: define secondary indices
-  // ...
```
Comment (simlu) [inline src/module/dy/create-model.js:51]: the dynamodb toolbox needs to know about these too!

**Review by simlu — APPROVED** [review]

**Pull request merged**